doc/README.gpu_support

```diff
@@ -0,0 +1,128 @@
+# Open-CE GPU Support
```

Comment:
This file should have an `md` extension like the other documents in this folder, otherwise GitHub won't render it properly.